src/components/chips/js/chipsController.js

```diff
@@ -225,8 +228,14 @@ MdChipsCtrl.prototype.getAdjacentChipIndex = function(index) {
  * call out to the md-transform-chip method, if provided.
  *
  * @param newChip
+ * @return {boolean|undefined} Returns true if the max chips limit is reached,
+ *   otherwise will always return undefined
  */
 MdChipsCtrl.prototype.appendChip = function(newChip) {
+  // Don't append the chip if the maxmimum is reached
+  // Return true, to show that the limit is reached
+  if (this.items.length >= this.maxChips) return true;
```

Comment:
I can swap the return values, it would make more sense to return `false` if the chip can't be added, but then I need to return `false` if the chip is a duplicate... and in this case it should clear the buffer. (so I think the current solution is the most simple)